components/badge/badge.css

```diff
@@ -6,6 +6,35 @@
     border-radius: 0.5rem;
 }
 
+.badge-icon {
+    font-size: 1.7em;
+    height:1.6em;
+    width:1.4em;
```

Comment:
maintain the consistency use rem (suggestion)

Reply:
acknowledged. thank you for the review @mikesch803